docs/application-development/observability.md

```diff
@@ -26,7 +26,7 @@ This section covers features related to viewing the state of the application, in
 - [Tracing](#tracing)
 - [Logging](#logging)
 - [Visibility](#visibility)
-- [Replays](#replays)
+- [Replays](/application-development/testing#replay)
```

Comment:
```suggestion
```

Oops, now that I see the context, it should instead be removed, as no longer in this section